zipline/utils/test_utils.py

```diff
@@ -115,15 +117,34 @@ def nullctx():
     Null context manager.  Useful for conditionally adding a contextmanager in
     a single line, e.g.:
 
-    with SomeContextManager() if some_expr else nullcontext:
+    with SomeContextManager() if some_expr else nullctx():
         do_stuff()
     """
     yield
 
 
+@contextmanager
+def security_list_copy():
+    old_dir = security_list.SECURITY_LISTS_DIR
+    new_dir = tempfile.mkdtemp()
+    try:
+        for subdir in os.listdir(old_dir):
+            shutil.copytree(os.path.join(old_dir, subdir),
+                            os.path.join(new_dir, subdir))
+            with patch.object(security_list, 'SECURITY_LISTS_DIR', new_dir), \
+                    patch.object(security_list, 'using_copy', True,
+                                 create=True):
+                yield
```

Comment:
When you decorate a method with `@contextmanager`, everything up until the yield becomes the eqiuvalent of the `__entering__` function for the context manager and everything after the yield becomes the `__exit__` function.